### PR TITLE
support django settings and pagination tags add  in templagetags

### PR DESCRIPTION
**Pull request by edison7500**
* support django settings 
*  bulma.css and font-awesome.css to use cdn 😄
* pagination tags add  in templagetags

### CODE REVIEW

File: bulma/bulma.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
```
Comment (timonweb) [inline bulma/bulma.py]: We support Python 3 only, this should be removed

File: bulma/bulma.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+from __future__ import unicode_literals
```
Comment (timonweb) [inline bulma/bulma.py]: We support Python 3 only, this should be removed

File: bulma/bulma.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+from importlib import import_module
```
Comment (timonweb) [inline bulma/bulma.py]: Were do we use this?

File: bulma/bulma.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+from django import VERSION as DJANGO_VERSION
```
Comment (timonweb) [inline bulma/bulma.py]: I don't think we need this too

File: bulma/bulma.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+from django.conf import settings
+
+
+# Do we support set_required and set_disabled?
+# See GitHub issues 337 and 345
+# TODO: Get rid of this after support for Django 1.8 LTS ends
+DBS3_SET_REQUIRED_SET_DISABLED = DJANGO_VERSION[0] < 2 and DJANGO_VERSION[1] < 10
```
Comment (timonweb) [inline bulma/bulma.py]: This comes from django-bootstrap I guess, we don't use it

File: bulma/bulma.py [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+
+BULMA_DEFAULTS = {
+    # The URL to the jQuery JavaScript file
+    'jquery_url': '//cdn.bootcss.com/jquery/1.12.4/jquery.min.js',
```
Comment (timonweb) [inline bulma/bulma.py]: I don't want to include any JS library there. We should get rid of jQuery

File: bulma/bulma.py [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    return BULMA.get(setting, default)
+
+def bulma_url(postfix):
+    """
+    Prefix a relative url with the bootstrap base url
+    """
+    return get_bulma_setting('base_url') + postfix
+
+
+def jquery_url():
```
Comment (timonweb) [inline bulma/bulma.py]: We should get rid of jquery

File: bulma/templates/bulma/pagination.html [hunk 8]
```diff
@@ -0,0 +1,42 @@
+{% load bulma_tags %}
+{% with bpurl=bulma_pagination_url|default:"" %}
```
Comment (timonweb) [inline bulma/templates/bulma/pagination.html]: We already have pagination, please see here: https://github.com/timonweb/django-bulma/blob/master/bulma/templates/pagination.html

File: bulma/bulma.py (hunk 5 above)
Comment (edison7500) [reply bulma/bulma.py]: yes, use django-bootstrap
my website  change from django-bootstrap to bulma

File: bulma/templates/bulma/pagination.html (hunk 8 above)
Comment (edison7500) [reply bulma/templates/bulma/pagination.html]: how to use it ? 
```
{% include pagination.html  with page_obj=xxxx %}

```

**Comment by NyanKiyoshi** [comment]
@timonweb still not good to merge?

File: bulma/templates/pagination.html [hunk 9]
```diff
@@ -1,26 +1,31 @@
...
-       href="?page={{ page_obj.next_page_number|stringformat:"d" }}{{ getvars }}{{ hashtag }}">{% trans "Next" %}</a>
-  {% else %}
-    <a class="pagination-next" disabled>{% trans "Next" %}</a>
-  {% endif %}
+      {% if current_page == num_pages %}
+          <a class="pagination-next" disabled>&raquo;</a>
+      {% else %}
+          <a class="pagination-next" href="{% bulma_url_replace_param bpurl parameter_name num_pages %}">&raquo;</a>
+      {% endif %}
+    <ul class="pagination-list">
```
Comment (NyanKiyoshi) [inline bulma/templates/pagination.html]: Could you keep a indentation of 2 spaces in the whole file? As we are using two spaces indentations in the other files.

**Pull request closed without merging**